package Ch22;

public class C01NullPointerException {
	public static void main(String[] args) {

		try {
			String data = null;
			System.out.println(data.toString());
			
			// Exception in thread "main" java.lang.NullPointerException
			//		at Ch22.C01NullPointerException.main(C01NullPointerException.java:6)

		} catch(NullPointerException e) {
			System.out.println("NullPointerException예외발생시 출력!!");
		}
		System.out.println("예외발생 이후 출력!!");
	}
}
